src/apps/xpubs/xpubs.py

```diff
@@ -69,10 +73,62 @@ async def menu(self, show_screen, show_all=False):
             if der is not None:
                 await self.show_xpub(der, show_screen)
                 return True
+        elif menuitem == 3:
+            format = await self.save_menu(show_screen)
+            if format is not False:
+
+                fingerprint = hexlify(self.keystore.fingerprint).decode()
+                extension = "json"
+                coin = net["bip32"]
+
+                derivations = [
+                    ('bip49', "p2wpkh", "m/49'/%d'/%d'" % (coin, self.account)),
+                    ('bip84', "p2sh-p2wpkh", "m/84'/%d'/%d'" % (coin, self.account)),
+                    ('bip44', "p2pkh", "m/44'/%d'/%d'" % (coin, self.account)),
+                    ('bip48_2', "p2wsh", "m/48'/%d'/%d'/2'" % (coin, self.account)),
+                    ('bip48_1', "p2sh-p2wsh", "m/48'/%d'/%d'/1'" % (coin, self.account)),
+                ]
+
+                if format == self.export_specter_diy:
+                    filedata = ""
+                    for der in derivations:
+                        xpub = self.keystore.get_xpub(der[2])
+                        filedata += "[%s/%s]%s\n" % (fingerprint, der[2], xpub.to_base58(net["xpub"]))
```

Comment:
Right now it writes `[fingerprint/m/deriavtion]xpub`, shoudl be `[fingerprint/derivation]xpub`. Also we noramlly use `h` for hardened derivation. We should use `'` only for compatibility with coldcard in json files.
```suggestion
                        filedata += "[%s/%s]%s\n" % (fingerprint, der[2][2:].replace("'","h"), xpub.to_base58(net["xpub"]))
```